pkg/cmd/provisioning/ibmcloud/create_shared_secrets.go

```diff
@@ -29,8 +29,8 @@ type: Opaque`
 	manifestsDirName = "manifests"
 )
 
-// APIKeyEnvVar is the environment variable name containing an IBM Cloud API key
-const APIKeyEnvVar = "IC_API_KEY"
+// APIKeyEnvVars is the environment variable names containing an IBM Cloud API key
+var APIKeyEnvVars = []string{"IC_API_KEY", "IBMCLOUD_API_KEY", "BM_API_KEY", "BLUEMIX_API_KEY"}
```

Comment:
Is there any kind of precedence criteria here? How ibm cloud cli handles it when all of these environment variables are set?

Reply:
go-client follows a same approach for reading the token here - 
https://github.com/IBM-Cloud/bluemix-go/blob/56edc36e39e52b243c4fb23a2cc63d8296d7d796/session/session.go#L41

Reply:
@mkumatag lmk what you come up with here. i'm guessing we should be consistent with the installer.

Reply:
if it is missing, we can enhance the code in the installer as well,